src/app/Http/Controllers/API/PollController.php

```diff
@@ -57,9 +77,13 @@ public function store(Request $request, Presentation $presentation)
      *
      * @param Poll $poll
      */
-    public function show(Poll $poll)
+    public function show(Request $request, Poll $poll)
     {
-        //
+        if ($request->user()->id != $poll->presenter() && $poll->state == State::DRAFT()) {
```

Comment:
one presentation could have many roles that can show/edit/create pools so you should use or create a function that returns all those users instead of one

Reply:
Fixed in dde2b70